themes/geekboot/assets/scss/dark-mode.scss

```diff
@@ -51,7 +51,7 @@
 
     // Override Bootstrap accordion colors in the body
     .accordion {
-        --bs-accordion-border-color: #{$fog-800};
+        --bs-accordion-border-color: #{$fog-500};
```

Comment:
There's something fishy going on with the bottom rounded corners of the accordions when they are in their expanded state - looks like it's still a dark color that doesn't match the straight line edge borders, or they are clipped or something? they look like they are missing essentially - do you see this too?

e.g. on https://deploy-preview-522--crossplane.netlify.app/v1.13/concepts/compositions/ 

![Screenshot 2023-08-08 at 10 08 16 AM](https://github.com/crossplane/docs/assets/4313439/5d972ce5-b256-49bf-bc69-c0622c404489)

Reply:
I see the same thing in dark mode.

Reply:
@jbw976 @plumbis I'll need to dig into that issue to find more. Being that issue is pretty minor and exists today, I may scope/investigate that in a separate PR if that works